Lesson5/main.js

```diff
@@ -0,0 +1,24 @@
+const ul = document.getElementById("js-ul");
+const fragment = new DocumentFragment();
+const features = [{to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+{to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}];
+
+const checkFeatures = new Promise((resolve) => {
+  resolve(features);
+}).then(
+  function(value){
```

Comment:
Comment
I think it's a good way!
You can write it in an Arrow function, but either is fine.
The following is written in Arrow functions.
```suggestion
const checkFeatures = new Promise((resolve) => {
  resolve(features);
}).then((value) =>{
```

Reply:
Thank you for suggestion!
Replaced standard function to arrow function, which makes my codes more consistent as I used arrow function in the promise state.